slides/linear_model.md

```diff
@@ -0,0 +1,260 @@
+class: titlepage
+
+.header[MOOC Machine learning with scikit-learn]
+
+# Linear Model
+
+This lesson covers the linear models. These are basic models, easy to understand and fast to train
+
+<img src="../scikit-learn-logo.svg">
+
+???
+
+Linear models are easy to understand and fast to train, 
+they give us fair baselines
+
+
+---
+# Outline
+
+* What is a linear model?
+* Linear model for regression & classification
+* How to avoid overfitting?
+
+
+---
+# Adult census
+
+.very-small[
+
+| Age | Workclass | Education    | Marital-status     | Occupation         | Relationship | Race  | Sex  | Capital-gain | Hours-per-week | Native-country | Salary |
+| --- | --------- | ------------ | ------------------ | ------------------ | ------------ | ----- | ---- | ------------ | -------------- | -------------- | ----- |
+| 25  | Private   | 11th         | Never-married      | Machine-op-inspct  | Own-child    | Black | Male | 0            | 40             | United-States  | $45k |
+| 38  | Private   | HS-grad      | Married-civ-spouse | Farming-fishing    | Husband     | White  | Male | 0            | 50             | United-States   | $40k |
+| 28  | Local-gov | Assoc-acdm   | Married-civ-spouse | Protective-serv    | Husband      | White | Male | 0            | 40             | United-States   | $60k  |
+| 44  | Private   | Some-college | Married-civ-spouse | Machine-op-inspct  | Husband      | Black | Male | 7688         | 40             | United-States   | $52k  |
+
+]
+Salary = *.4 x* education + *.2 x* hours-per-week + *.1 x* age + ...
+
+???
+Adult census is here a bit modify, instead we have the value of the salary for every one.
+So we can see it as a regression problem
```

Comment:
```suggestion
Adult census is here a bit modify, instead of having 2 categories, < $50k and
>= $50k, the target "Salaray" contains the exact value of the salary for each
person. Thus, the target is continuous and we deal with a regression problem
instead of a classification problem.
```